contributing/en/create-and-edit.md

```diff
@@ -99,8 +99,8 @@ New assets can be added before working on a document or after it has been comple
 
 To add assets to a document and assets folder:
 
-1. Add image code in the part of the document you would like this to appear in. To link to an image in the assets folder you will need to type `“![](./assets/ image file name)”`
-    e.g., `![](./assets/learning-ojs3.1-jm-settings-workflow-comp.png)`
+1. Add image code in the part of the document you would like this to appear in. To link to an image in the assets folder you will need to type `“![alt text](./assets/ image file name)”`
+    e.g., `![alt text](./assets/learning-ojs3.1-jm-settings-workflow-comp.png)`
```

Comment:
@librariam Instead of saying "alt text", should this have an example of what the actual alt text would be?

Reply:
Sure, I added a couple examples with images further down in the "alt text" section but didn't think to add actual examples elsewhere. This makes sense. Would you like to add one here? Without seeing the image, I'd put something like [OJS 3.1 workflow settings components tab.]

Reply:
Maybe instead it would make sense to link to the section further down that has the examples? I can add that if you think it makes sense here.